packages/sit-onyx/src/components/OnyxDataGrid/features/all.ts

```diff
@@ -3,6 +3,7 @@ export * from "./sorting/types";
 export * from "./stickyColumns/types";
 
```

Comment:
Don't forget to also export the types so that the users are able to use them:
```suggestion
export * from "./hideColumns/types";

```